Library/Homebrew/cmd/audit.rb

```diff
@@ -1235,3 +1232,31 @@ def problem(text)
     @problems << text
   end
 end
+
+# Debugging support
```

Comment:
Yes. In fact, we can probably just call `inject_dump_stats!` since it looks like the same functionality. Thanks for pointing that out.